9.2/run-postgresql.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -e
+
+# Be paranoid and stricter than we should be.
+psql_identifier_regex='^[a-zA-Z_][a-zA-Z0-9_]*$'
+psql_password_regex='^[a-zA-Z0-9_~!@#$%^&*()-=<>,.?;:|]+$'
+
+function usage {
+	echo "You must specify following environment variables:"
+	echo "  \$POSTGRESQL_USER (regex: '$psql_identifier_regex')"
+	echo "  \$POSTGRESQL_PASSWORD (regex: '$psql_password_regex')"
+	echo "  \$POSTGRESQL_DATABASE (regex: '$psql_identifier_regex')"
```

Comment:
I think we discussed somewhere for MySQL image that we don't want optional vars and that we recommend that in the image writing guide, that's why it's not optional.

Personally, I'd be in favour of that, it would make using the image outside Openshift a LOT easier.